schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+CREATE TABLE animals (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name CHAR(255),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BIT,
+    weight_kg DECIMAL(5, 2),
```

Comment:
- [ ] Please kindly pay attention to line 7 (neutered column), I couldn't get answers to the queries because you used different name for this column in **`data.sql`**. More so, the data type you assigned in **`data.sql`** (**BOOLEAN**) is not in line with what was declared here (**BIT**). I will suggest that you use **BOOLEAN** for the data type as stated in the requirement. 🙏